jwst/pipeline/cube_build.cfg

```diff
@@ -1,16 +1,3 @@
 name = "cube_build"
 class = "jwst.cube_build.CubeBuildStep"
-channel = "all"
-band = "all"
-filter = "all"
-grating = "all"
-scale1 = 0.0
-scale2 = 0.0
-scalew = 0.0
-weighting = "emsm"
-coord_system = 'skyalign'
-rois = 0.0
-roiw = 0.0
 weight_power = 2.0
```

Comment:
A reason to keep `weight_power` here?  This is different from the default in the step.  Perhaps the default in `CubeBuildStep.spec` should be changed and this removed?

Reply:
I left it in for now because it *is* different than the default and couldn't tell if that was intentional or not. The default in the step spec is zero. @jemorrison or @drlaw1558 can either of you tell us which is the preferred default (0 or 2)?

Reply:
Hm, 2 should be the default, as this is 1/r^2 MSM weighting.  However, the difference is less important at the moment since EMSM is the current default weighting scheme so the MSM parameters are less likely to be used.

Reply:
Thanks for the confirmation. So in this case the value in the cfg file is correct, but the default in the step code is not. So we should fix the default value in the step code and then remove this one remaining param from the cfg file.